src/main/webapp/css/timeline.css

```diff
@@ -45,6 +47,66 @@
   text-align: left;
 }
 
+.timeline > li .timeline-panel{
+  background-color: rgba(255, 255, 255, 0.6);
+  border-radius: 0.5em;
+}
+
+.timeline .delete {
+  float: right;
+  margin-right: -19px;
+  font-size: 1.5em;
+  color: rgba(255, 146, 52, .8);
+  padding: 0;
+  background-color: rgba(255,255,255,0);
+  border-style: hidden;
+}
+
+.timeline .delete:hover, .timeline .delete:active{
+  color: rgba(229, 127, 39, 0.87);
+}
+
+.deleteConfirmation {
+  position: absolute;
+  width: 100%;
+  left: 0;
+  text-align: center;
+  transition: opacity 0.1s ease;
+  -webkit-transition: opacity 0.1s ease;
+  -moz-transition: opacity 0.1s ease;
+  font-family:sans-serif;
+  background-color:#29130dcf;
```

Comment:
Maybe we should use `rgba` just so its a little more readable and easy to change? I also personally think this alert could be slightly less opaque, something like `rgba(41, 19, 13, 0.9)`?